handlers/artists.go

```diff
@@ -130,7 +130,7 @@ func registerArtists(app *pocketbase.PocketBase) {
 
 					row := map[string]any{
 						"Name":       m.GetString("name"),
-						"Url":        artistUrl(m.GetString("slug")),
+						"Url":        slugUrl("/artists/", m.GetString("slug")),
```

Comment:
Why was this change implemented?

Reply:
To be honest, i do not remember anymore. Probably i wanted to use this also at one time, and made it more general to avoid code duplication. Although I do not use it in musics but i would keep this change